### PR TITLE
RHTAP: Add inputRefs to CloudWatch log forwarder

### PR DESCRIPTION
**Pull request by eisraeli**
Add inputRefs to CloudWatch log forwarder as they were prematurely, as we still don't have those audit logs in Splunk.

The intension here is to have also `infrastructure` and `audit` logs in CloudWatch for the time being.

### CODE REVIEW

**Comment by Omeramsc** [comment]
LGTM

**Review by amfred — CHANGES_REQUESTED** [review]
I don't want to re-enable this, because it will cost us a lot of money in AWS fees, and sending the audit logs to Splunk is already done for us as part of our ROSA subscription.  Here's the reference document: 

https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.html

“Red Hat site reliability engineers (SREs) maintain a centralized monitoring and alerting system for all ROSA cluster components, the SRE services, and underlying AWS accounts. **Platform audit logs** are securely forwarded to a centralized security information and event monitoring (SIEM) system, where they may trigger configured alerts to the SRE team and are also subject to manual review. Audit logs are retained in the SIEM system for one year. Audit logs for a given cluster are not deleted at the time the cluster is deleted.”

Basically, we're already paying for that with our ROSA subscription.

What are you trying to accomplish?
